spec/features/topical_event_spec.rb

```diff
@@ -56,12 +51,15 @@
     expect(page).to have_link(content_item.dig("details", "about_page_link_text"), href: "#{base_path}/about")
   end
 
-  context "when there are social media links" do
-    it "includes links to the social media accounts" do
-      visit base_path
-      expect(page).to have_link("Facebook", href: "https://www.facebook.com/a-topical-event")
-      expect(page).to have_link("Twitter", href: "https://www.twitter.com/a-topical-event")
-    end
+  it "includes a link to the organisation" do
```

Comment:
Would it be possible to have a test that covers the scenario when there are no organisations (and therefore tests that the header isn't included)?

Also a test for the organisation logos section would be good too.

Reply:
👍